client/landing/jetpack-cloud/index.js

```diff
@@ -1,25 +1,84 @@
 /**
  * External dependencies
  */
-import config from '../../config';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
-import initJetpackCloudRoutes from './routes';
-import { bootApp } from 'boot/common';
 
-/**
- * Style dependencies
- */
-import 'components/environment-badge/style.scss';
-import 'layout/style.scss';
-import 'assets/stylesheets/jetpack-cloud.scss';
-
-window.AppBoot = () => {
-	if ( ! config.isEnabled( 'jetpack-cloud' ) ) {
-		window.location.href = '/';
-	} else {
-		bootApp( 'Jetpack Cloud', initJetpackCloudRoutes );
-	}
-};
+import { navigation, siteSelection, sites } from 'my-sites/controller';
```

Comment:
Let's reconsider what we reuse from the my sites controller. It doesn't seem like we need the `navigation` one because we're completely replacing it later. I'm not 100% sure we need the `sites one the way it is right now, too. It will need some customization either way, so should we just create a Jetpack Cloud version of it?

Reply:
> The navigation function is needed because it sets context.secondary to a MySitesNavigation component. Without it there is no site selection in the sidebar.

This doesn't sound like it makes a whole lot of sense. My gut instinct is that site selection and site navigation component should be two independent items handled separately. Also, ideally, because here we have our own sidebar, we should have a different version of `navigation`.